nautobot/docs/administration/nautobot-server.md

```diff
@@ -129,10 +129,9 @@ nautobot=> \q
 
 ### `dumpdata`
 
-<!-- markdownlint-disable MD036 -->
-_Changed in version 1.3.0: `extras.job` should now be included (removed `--exclude extras.job`)_
-_Changed in version 1.3.0: `django_rq` should now be excluded (add `--exclude django_rq`)_
- <!-- markdownlint-enable MD036 -->
++/- 1.3.0
```

Comment:
```suggestion
+/- 1.3.5
```

https://nautobot.readthedocs.io/en/stable/release-notes/version-1.3/#fixed_5
#1786

Reply:
1.3.5 is where the documentation was updated, but 1.3.0 is the point at which this recommendation became relevant.